docs/user/05-50-configure-list-of-modules.md

```diff
@@ -0,0 +1,76 @@
+# Configure list of modules
+
+By default, Kyma Environment Broker applies the Kyma CR, including the default modules selected by Kyma, to a cluster.
+You can configure your custom list of modules by setting the "modules" object in the JSON schema.
```

Comment:
```suggestion
You can configure your custom list of modules by setting the **modules** object in the JSON schema.
```
I'm not sure about this, but I guess we could use bold font for "modules" and "oneof". I think they're both JSON Schema keywords (for example, see this [doc](https://json-schema.org/blog/posts/applicability-json-schema-fundamentals-part-1#putting-everything-together-avoiding-oneof-pitfalls))

Reply:
Thank you for clarifying that